src/pagination/pagination.spec.ts

```diff
@@ -145,6 +150,45 @@ describe('ngb-pagination', () => {
     beforeEach(
         () => { TestBed.configureTestingModule({declarations: [TestComponent], imports: [NgbPaginationModule]}); });
 
+    it('should render and respond to collectionSize change with customPages', () => {
+      const html = `<ngb-pagination [collectionSize]="collectionSize" [page]="1" [pageSize]="pageSize">
```

Comment:
1. a **minimal** test case would be better... ex. I don't think we case about aria attibutes, classes, etc
2. instead of repeating the same template for 4 tests I think I would have extracted a `describe` block for this with a common test component